src/components/views/rooms/RoomHeader.js

```diff
@@ -32,6 +32,9 @@ import RoomTopic from "../elements/RoomTopic";
 import RoomName from "../elements/RoomName";
 import {PlaceCallType} from "../../../CallHandler";
 import {replaceableComponent} from "../../../utils/replaceableComponent";
+import exportConversationalHistory from '../../../utils/exportUtils/exportUtils';
+import { exportFormats, exportOptions } from '../../../utils/exportUtils/exportUtils';
```

Comment:
Would be important to async-import these somewhere along the chain so that every app load doesn't demand jszip and streamsaver

In this place we could rely on a Modal for managing the export settings, and that modal could be async-imported, with everything there-in being normally imported to simplify things

Reply:
Yes, definitely. This button placement is very temporary, it will be moved once we get the design